cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -1171,21 +1200,27 @@ Found {{ len data }} Databricks compute resources that were *idle* during the la
       field "savings" do
         label "Estimated Savings"
       end
+      field "savingsCurrency" do
+        label "Currency"
+      end
       field "recommendationDetails" do
         label "Recommendation"
       end
-      field "subscriptionId" do
+      field "accountID" do
         label "Subscription ID"
         path "databricks_workspace.subscriptionId"
       end
-      field "subscriptionName" do
+      field "accountName" do
```

Comment:
When changing this field (the same with `accountID`) you must ensure that there is a previous reference to this field, otherwise this will show as empty in the incident because `accountName` does not exist.

It is still referenced as `subscriptionName` everywhere else in the policy, so just doing this change and not updating the code elsewhere will not work.

Reply:
Actually nvm, this should be fine because it is referencing a path, of which is unchanged.